### ISSUE TITLE
Night Mode 3.6.0 

### ISSUE THREAD

**Issue by jonandersen** [plaid/plaid-link-android#197]
# The problem

Using the SDK in night mode doesn't render properly. Works fine in Day mode. 

# Environment

|                        |  |
| ------------------------| ---------- |
| Android OS Version                    | e.g. 11.0.0 (30)                |
| Android Devices/Emulators    | e.g. Pixel 3a physical                |

# Steps to Reproduce
* Open plaid SDK 
* Tap "Continue"
* Bank select screen list items renders strangely 

# Expected Result
The sdk to work in night mode (or disable support)

# Screenshots
![Screenshot_20210930-150841](https://user-images.githubusercontent.com/1019479/135473141-d3d51a91-1eef-45fc-85a2-a88326bac280.png)

**Comment by amytang0** [comment]
Just to confirm, did you opt into the native beta? This will be fixed in `3.6.1` and can be confirmed by using `3.6.1-rc.1`

**Comment by jonandersen** [comment]
@amytang0 yep, we did opt in for that. Looks great besides the above bug!

**Comment by amytang0** [comment]
3.6.1 is now officially out and should resolve this bug. Please re-open if this is not the case, thank you!

**Comment by Heatherlee85527** [comment]
Yes I did and I'm trying to find out in things I had identity theft I really Need help tracking
